.github/workflows/test.yml

```
@@ -0,0 +1,50 @@
+# documentation: https://help.github.com/en/articles/workflow-syntax-for-github-actions
+name: Tests for EESSI test suite, using EESSI pilot repo
+on: [push, pull_request, workflow_dispatch]
+permissions: read-all
+jobs:
+  test_with_eessi_pilot:
+    runs-on: ubuntu-22.04
+    strategy:
+      fail-fast: false
+      matrix:
+        EESSI_VERSION:
+        - "2021.12"
+    steps:
+        - name: Check out software-layer repository
+          uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # v3.1.0
+          with:
+            persist-credentials: false
+
+        - name: Mount EESSI CernVM-FS pilot repository
+          uses: eessi/github-action-eessi@58b50fd2eead2162c2b9ac258d4fb60cc9f30503  # v2.0.13
```

```suggestion
          uses: eessi/github-action-eessi@58b50fd2eead2162c2b9ac258d4fb60cc9f30503  # v2.0.13
          with:
            eessi_stack_version: ${{matrix.EESSI_VERSION}}
```
with this change, no  need for the step to set up the EESSI environment

---

fixed in 785b2eb